src/api/order/email/domain/order-email-domain.v6.service.js

```diff
@@ -0,0 +1,44 @@
+angular.module('ovh-api-services').service('OvhApiOrderDomainV6', ($resource, $cacheFactory) => {
```

Comment:
```suggestion
angular.module('ovh-api-services').service('OvhApiOrderEmailDomainV6', ($resource, $cacheFactory) => {
```
else there could be some conflits with the `/order/domain` route